env.yml

```diff
@@ -0,0 +1,25 @@
+name: op_env
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - python=3.9
+  - pip
+  - pandas
+  - numpy
+  - pytorch
+  - matplotlib
+  - seaborn
+  - scikit-learn
+  - jupyterlab
+  - torchvision
+  - tqdm
+  - opencv
+  - scikit-image
+  - pip:
+    - git+https://github.com/facebookresearch/segment-anything.git
```

Comment:
I do not see CellProfiler being installed in this env (which I can understand why if there are more issues from install haha).

Since you use CellProfiler in this PR, I do not see where you have installed it in a different environment or if you use the web binary install. I would recommend to include these instructions as it is hard to reproduce these results without knowing where and how CellProfiler is installed.

Reply:
I added cell profiler to the env file!